### PR TITLE
Chore: Move internal GraphNG+Timeseries components into core

### PR DESCRIPTION
**Pull request by ryantxu**
**What is this feature?**

This moves as much <GraphNG .../> things to core as possible -- this is mainly internal components that are are in @grafana/ui for legacy reasons (and mostly not exposed or usable!)

**Why do we need this feature?**

The circular dependencies of needing add more things to grafana/ui is really complicated.  This will simplify that dev experience

**Who is this feature for?**

[Millionaires (obviously).](https://github.com/grafana/grafana/issues/77409)

TODO:
- [x] Get build fully working (with breaking changes)
- [x] Add back GraphNG components to avoid breaking changes, but with clear deprecations and removal notice

Would be nice to also move the uPlot setup -- it is mostly internal, but the impact on Sparkline (and by extension, table) gets complicated :(

### CODE REVIEW

**Review by nmarrs — APPROVED** [review]
this approach is reasonable to me - I would be good with merging this first and then having a follow-up PR for re-adding deprecated code back to grafana-ui

**Comment by nmarrs** [comment]
> this approach is reasonable to me - I would be good with merging this first and then having a follow-up PR for re-adding deprecated code back to grafana-ui

Scratch that - just heard from @joshhunt that every release to `main` grafana/ui is also included as part of that and therefore we should address re-adding the missing functions back to grafana/ui with deprecations as part of this PR

**Comment by ryantxu** [comment]
@nmarrs -- i updated this so we now have a "graveyard" folder, and the exports are the same as before.  `[levitate]` seems happy, so that is good!

File: .betterer.results [hunk 1]
```diff
@@ -1032,6 +1010,39 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
+    "packages/grafana-ui/src/graveyard/GraphNG/GraphNG.tsx:5381": [
```
Comment (ryantxu) [inline .betterer.results:1013]: sorry... this doubles the betterer errors, but at least half are in a graveyard ☠️ 💀 ☠️

**Review by nmarrs — APPROVED** [review]
🪦

**Review by leeoniya — APPROVED** [review]
tested a bunch of `graph-ng` gdev dashboards, all seem to work.

thanks for doing this :heart:!

File: .betterer.results (hunk 1 above)
Comment (ashharrison90) [reply .betterer.results:1013]: @ryantxu what's the plan for `/graveyard`? i'm guessing we remove at some point in the future... what is that point? we could ignore `/graveyard` in betterer (that's what we do for `/angular` for example)

**Pull request merged**